### ISSUE TITLE
Build failed:undefined: forward.NewProxy

### ISSUE THREAD

**Issue by saiteTimedOut** [coredns/alternate#26]
Build failed with master branch(06cd8439182be8b9ed17280eaae411f9afb7c8f2),the plugin may need a update?
`# github.com/coredns/alternate
../../../go/pkg/mod/github.com/coredns/alternate@v0.0.0-20201105225029-f0d10f2aa3aa/forward.go:43:16: undefined: forward.NewProxy
`

**Comment by pixhub** [comment]
Same issue here

**Comment by chrisohaver** [comment]
Yes. Proxy moved to separate “proxy” package in coredns master (to be 1.11.0). All plugins that reference it will need an update to use latest coredns.
